app/src/components/buttons/ghostButton/ghostButton.scss

```diff
@@ -230,6 +230,9 @@
       background: transparent;
       color: $COLOR--topaz-hover;
     }
+    &.transparent-less-width {
+      min-width: 90px;
+    }
```

Comment:
What the purpose to create a new class and manage it with prop? Seems like we can add property directly to the `transparent-background` class.